h5f_file.go

```diff
@@ -106,36 +106,36 @@ func (f *File) Flush(scope Scope) error {
 // FIXME
 // Retrieves name of file to which object belongs.
 func (f *File) FileName() string {
-	// ssize_t H5Fget_name(hid_t obj_id, char *name, size_t size )
+	// size_t H5Fget_name(hid_t obj_id, char *name, size_t size )
```

Comment:
I can't remember why these are here. I think the signature comments can go away.